<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Saber Music</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>res/pics/saber.ico</normaloff>res/pics/saber.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>130</y>
      <width>291</width>
      <height>671</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#textEdit
{
border-radius:25px;
border:1px solid rgb(222,223,169);
background:rgba(255, 255, 255, 200)
}</string>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="markdown">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>740</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#comboBox{
border:1px solid rgb(188, 188, 188);
border-radius:10px;
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1300</width>
      <height>850</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>res/pics/bg.png</pixmap>
    </property>
   </widget>
   <widget class="QFrame" name="horizontalFrame">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>40</y>
      <width>221</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="hide">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#hide{
border: none;}

#hide:hover {
padding-bottom:5px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>res/pics/hide.png</normaloff>res/pics/hide.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="minSize">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#minSize{
border: none;}

#minSize:hover {
padding-bottom:5px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>res/pics/minsize.png</normaloff>res/pics/minsize.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#close{
border: none;}

#close:hover {
padding-bottom:5px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>res/pics/close.png</normaloff>res/pics/close.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>45</y>
      <width>71</width>
      <height>67</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton{
border: 1px solid #dcdfe6;
padding: 10px;
border-radius: 5px;}

#pushButton:hover {
background-color: rgb(255,255,209);
border: 1px solid rgb(222,223,169);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>res/pics/search.png</normaloff>res/pics/search.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="AboutButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#AboutButton{
border:none;
padding:2px;
}
#AboutButton:hover {
border: 2px solid rgb(44, 44, 44,100);
border-radius:5px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>res/pics/saber.png</normaloff>res/pics/saber.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>UD Digi Kyokasho NP-R</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Saber Music</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>45</y>
      <width>500</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>470</width>
      <height>70</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit
{
border-radius:20px;
background:rgba(0,0,0, 100);
color:white;
border-bottom:1px solid rgb(222,223,169);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>搜索......</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>851</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正姚体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane{
min-width:70px;
min-height:25px;
border-top: 2px solid;
border-radius:20px;
background:rgba(255, 255, 255, 200)

}

QTabBar::tab {

min-width:70px;

min-height:25px;

color: white;

font: 15pt &quot;幼圆&quot;;
border: 0px solid;



}

QTabBar::tab:selected{

min-width:70px;

min-height:25px;
color: white;

	font: 12pt &quot;幼圆&quot;;

border: 0px solid;

border-bottom: 2px solid;

border-color: #4796f0;

}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>搜索</string>
     </attribute>
     <widget class="QTreeView" name="treeView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>851</width>
        <height>571</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#treeView
{
border-radius:20px;
border:2px solid rgb(222,223,169);
background:rgba(255, 255, 255, 0)
}</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>喜爱</string>
     </attribute>
     <widget class="QTreeView" name="treeView_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>851</width>
        <height>571</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#treeView_2
{
border-radius:20px;
border:2px solid rgb(222,223,169);
background:rgba(255, 255, 255, 0)
}</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>730</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="loveButton">
       <property name="styleSheet">
        <string notr="true">#loveButton
{
border:none;
}

#loveButton:hover{
padding-bottom:5px;
}

#loveButton:pressed
{
padding-top:5px;
padding-left:5px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>res/pics/love.png</normaloff>res/pics/love.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="styleSheet">
        <string notr="true">#browseButton
{
border:none;
}

#browseButton:hover{
padding-bottom:5px;
}

#browseButton:pressed
{
padding-top:5px;
padding-left:5px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>res/pics/browser.png</normaloff>res/pics/browser.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>bg</zorder>
   <zorder>textEdit</zorder>
   <zorder>comboBox</zorder>
   <zorder>horizontalFrame</zorder>
   <zorder>pushButton</zorder>
   <zorder>AboutButton</zorder>
   <zorder>title</zorder>
   <zorder>lineEdit</zorder>
   <zorder>tabWidget</zorder>
   <zorder>horizontalLayoutWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>minSize</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1097</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>1064</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1166</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>1158</x>
     <y>-19</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
